documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -256,6 +256,42 @@ The following example shows a data change event with ordered transaction metadat
 }
 ----
 
+[[vitess-efficient-transaction-metadata]]
+=== Efficient Transaction Metadata
+
+If you enable the connector to provide transaction metadata, it generates significantly more data, including the number of messages sent, and the size of each message.
```

Comment:
```suggestion
If you enable the connector to provide transaction metadata, it generates significantly more data. It will send additional messages to the transaction topics and the size of each message sent to the data change topics will be increased (because it includes the transaction metadata block).
```